<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #161616;
</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnBedroom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKitchen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLivingRoom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDanaOffice">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDanielOffice">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStudio">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAllOn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAllOff">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAway">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #3a3a3a</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnParty">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>21</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #3a3a3a</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="brightnessSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #3f3f3f;
    margin: 2px 0;
}

QSlider::groove:horizontal:disabled {
	background: #313131;
	border-radius: 2px;
	border-style: none;
}

QSlider::add-page {
	background: #3f3f3f;
	border-radius: 3px;
}

QSlider::sub-page {
	background: #f0f0f0;
	border-radius: 3px;
}

QSlider::sub-page:disabled {
	background: #6d6d6d;
	border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #9557c6;
    width: 30px;

    margin: -20px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}

QSlider::handle:horizontal:disabled {
	 background: #3a3a3a;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>254</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColorMode1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColorMode2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColorMode3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColorMode4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectedOff">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectedOn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #161616;
	border-width: 3px;
	border-color: #3a3a3a;
	border-style: outset;
	border-radius: 15px;
}

QPushButton:pressed{
	background-color: #ffffff;
}

QPushButton:checked{
	background-color: #ffffff;
	border-style: inset;
	border-color: #9557c6;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
